docs/user_guides/debugging/local_variables.rst

```diff
@@ -4,89 +4,94 @@ Local variables
 
-
 ``info locals``
 ---------------
 
 Run debugger:
 
 .. code-block:: bash
 
-    export NUMBA_DPPY_DEBUG=1
+    export NUMBA_DPPY_DEBUGINFO=1
     export NUMBA_OPT=0
-    gdb-oneapi -q --args python local_vars.py
-    (gdb) break local_vars.py:9
-    No source file named local_vars_ex.py.
+    gdb-oneapi -q --args python sum.py
+    (gdb) break sum.py:8
+    No source file named sum.py.
     Make breakpoint pending on future shared library load? (y or [n]) y
-    Breakpoint 1 (local_vars_ex.py:9) pending.
+    Breakpoint 1 (sum.py:8) pending.
     (gdb) run
     (gdb) info locals
 
-GDB output:
+GDB output ``NUMBA_OPT=0``:
 
 .. code-block:: bash
 
-    Thread 2.2 hit Breakpoint 1, with SIMD lanes [0-7], dppy_py_devfn__5F__5F_main_5F__5F__2E_data_5F_parallel_5F_sum_24_1_2E_array_28_float32_2C__20_1d_2C__20_C_29__2E_array_28_float32_2C__20_1d_2C__20_C_29__2E_array_28_float32_2C__20_1d_2C__20_C_29_ () at local_vars.py:9
-    9           c[i] = a[i] + b[i]
-    (gdb) info locals
-    a = '\000' <repeats 55 times>
-    b = '\000' <repeats 55 times>
-    c = '\000' <repeats 55 times>
-    __ocl_dbg_gid0 = 0
-    __ocl_dbg_gid1 = 0
-    __ocl_dbg_gid2 = 0
+    Thread 2.1 hit Breakpoint 1, with SIMD lanes [0-7], __main__::data_parallel_sum () at sum.py:8
+    8         i = dppy.get_global_id(0)
+
+    a_in_kernel = "\340\354\205YUU\000\000\340\354\205YUU\000\000\001", '\000' <repeats 11 times>, "\001\000\000\000\000\000\000\000UU\000\000 \355\205YUU\000\000 \355\205YUU\000"
+    b_in_kernel = "\240\357\205YUU\000\000\001", '\000' <repeats 11 times>, "\001\000\000\000\000\000\000\000\377\177\000\000\340\357\205YUU\000\000\340\357\205YUU\000\000\001\000\000\000\000\000\000"
+    c_in_kernel = "\001", '\000' <repeats 11 times>, "\001\000\000\000\000\000\000\000UU\000\000М\205YUU\000\000М\205YUU\000\000\001", '\000' <repeats 11 times>, "\001\000\000"
+    i = 1
+    l1 = 0
+    l2 = 0
+    __ocl_dbg_gid0 = 4
+    __ocl_dbg_gid1 = 2
+    __ocl_dbg_gid2 = 2
     __ocl_dbg_lid0 = 0
-    __ocl_dbg_lid1 = 0
-    __ocl_dbg_lid2 = 93825017857072
-    __ocl_dbg_grid0 = 0
+    __ocl_dbg_lid1 = 7
+    __ocl_dbg_lid2 = 0
+    __ocl_dbg_grid0 = 5
     __ocl_dbg_grid1 = 0
     __ocl_dbg_grid2 = 0
 
 .. note::
 
     Known issues:
-      - Representation of local variables values is currently not available.
+      - Debugger can show the variable values, but these values may not match the actual value of the referred variables.
 
 
 ``print variable``
 ------------------
 
 .. code-block:: bash
 
-    (gdb) print a
-    $1 = '\000' <repeats 55 times>
+    (gdb) print a_in_kernel
+    $1 = "\340\354\205YUU\000\000\340\354\205YUU\000\000\001", '\000' <repeats 11 times>, "\001\000\000\000\000\000\000\000UU\000\000 \355\205YUU\000\000 \355\205YUU\000"
+
+    (gdb) print i
+    $2 = 1
+
 
 .. note::
 
     Known issues:
-      - Kernel variables are shown in IR representation.
+      - Kernel variables names are shown in IR representation.
```

Comment:
```suggestion
      - Kernel variable names are shown in IR representation.
```

Reply:
Is it clear to a reader, what is "IR representation"? If not providing an explanation in this page, perhaps we could add a link to a page which explains / shows it to the user?